packages/plugin-commands-installation/src/fetch.ts

```diff
@@ -0,0 +1,72 @@
+import { docsUrl } from '@pnpm/cli-utils'
+import { UNIVERSAL_OPTIONS } from '@pnpm/common-cli-options-help'
+import { Config, types as allTypes } from '@pnpm/config'
+import { createOrConnectStoreController, CreateStoreControllerOptions } from '@pnpm/store-connection-manager'
+import { InstallOptions, mutateModules } from 'supi'
+import * as R from 'ramda'
+import renderHelp from 'render-help'
+
+export const rcOptionsTypes = cliOptionsTypes
+
+export function cliOptionsTypes () {
+  return R.pick([
+    'production',
+    'dev',
+  ], allTypes)
+}
+
+export const commandNames = ['fetch']
+
+export function help () {
+  return renderHelp({
+    description: 'Fetch packages from a lockfile into virtual store, package manifest is ignored',
```

Comment:
Might be good to mark it as experimental for now.
```suggestion
    description: 'Fetch packages from a lockfile into virtual store, package manifest is ignored. WARNING! This is an experimental command. Breaking changes may be introduced in non-major versions of the CLI',
```